frontend/public/components/secrets/create-secret.tsx

```diff
@@ -70,6 +70,10 @@ const generateSecret = () => {
   return s4() + s4() + s4() + s4();
 };
 
+const generateUID = (prefix) => {
+  return _.uniqueId(prefix);
+};
```

Comment:
I think my comment got lost earlier, but I don't see a reason to wrap `_.uniqueId`. We're just calling it with the same parameter.